.github/workflows/run-checks-github-runners.yml

```diff
@@ -18,6 +18,11 @@ env:
   # Can't use ${{ env.REGISTRY }} here, but would like to, so that we' don't
   # repeat ghcr.io.
   IMAGE_TAG: ghcr.io/${{ github.repository }}:sha-${{ github.sha }}
+  
+concurrency:
+  # Cancel in-progress runs for branches other than main  
```

Comment:
I don't think this comment is exactly correct. This doesn't necessarily _ignore_ the main branch. It just uses a different group ID if the branch is main (it uses run_id if the branch is main). This basically means that this workflow won't get canceled for pushes to main, which is fine for our purposes.

Can you add a reference to the source where you took the code from?
https://www.meziantou.net/how-to-cancel-github-workflows-when-pushing-new-commits-on-a-branch.htm

Thank you!

Reply:
To be clear, please:

- [ ] Update the comment to reflect what the concurrency group actually does
- [ ] Add a reference to the link